src/App.js

```diff
@@ -14,11 +14,76 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+const icons = [
+  '🥟',
+  '🥕',
+  '🥚',
+  '🍦',
+  '🍏',
+  '🍞',
+  '🍥',
+  '🍯',
+  '🧉',
+  '🧄',
+];
+
+class App extends React.Component {
+  state ={
+    selectedGood: ['Jam'],
+  }
+
+  render() {
+    const { selectedGood } = this.state;
 
+    return (
+      <div className="App">
+        <h1 className="App__header">
+          {selectedGood.length > 0
+            ? (`${selectedGood} is selected`)
+            : 'No goods are selected'
+        }
+          {this.state.selectedGood.length > 0
+            && (
```

Comment:
```suggestion
          {!!selectedGood.length && (
```
You have used destructuring a couple of lines above. No need to write `this.state` here